client/src/Auth/components/SignUp.js

```diff
@@ -0,0 +1,121 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+const SignUp = ({ handleClickOther }) => {
+  const [userText, setUserText] = useState('');
+  const [passText, setPassText] = useState('');
+  const [confPassText, setConfPassText] = useState('');
+
+  const handleUserChange = (e) => {
+    setUserText(e.target.value);
+  };
+
+  const handlePassChange = (e) => {
+    setPassText(e.target.value);
+  };
+
+  const handleConfPassChange = (e) => {
+    setConfPassText(e.target.value);
+  };
+
```

Comment:
@cgf5033 seeing all of these inputs with the same requirements and internal state make me wonder if we can offload the functionality into a separate and reusable component. Not super high on the docket though.